# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/generic/HyperTextAccessible.h
# Commit: c0a7fdc4bfca
# Full Hash: c0a7fdc4bfca0596ffecd347f0e248440e371989
# Author: Eitan Isaacson <eitan@monotonous.org>
# Date: 2022-01-05 09:33:53
# Regressor Bug: 1737944
# File Overlap Count: 2
# Description:
#   Bug 1737944 - P5: Unify GetLevelInternal and GetDefaultLevel as Accessible::GetLevel. r=Jamie
#   
#   The nsAccUtils method was a "fast" one for calculating set sizes and
#   conceptual parents. Unified it with
#   LocalAccessible::GetLevelInternal in Accessible::GetLevel with an
# ==============================================================================

diff -r f6ce0e05c92e -r c0a7fdc4bfca accessible/generic/HyperTextAccessible.h
--- a/accessible/generic/HyperTextAccessible.h	Tue Jan 04 21:01:35 2022 +0000
+++ b/accessible/generic/HyperTextAccessible.h	Tue Jan 04 21:01:36 2022 +0000
@@ -51,7 +51,6 @@
 
   // LocalAccessible
   virtual nsAtom* LandmarkRole() const override;
-  virtual int32_t GetLevelInternal() override;
   virtual already_AddRefed<AccAttributes> NativeAttributes() override;
   virtual mozilla::a11y::role NativeRole() const override;
   virtual uint64_t NativeState() const override;